### PR TITLE
Load amsterdam style

### PR DESCRIPTION
**Pull request by ghost**
Amsterdam style repo installed as dependency and loaded/copied during amaps build. Together with nlmaps@^2.3.16, the demo viewer will now use this styling.

Copied to `test/dist` in `build-amaps-dev`, and to `dist` in `build-amaps`.

The index.html has also been adjusted to load `dist/css/ams-stijl.css`.

### CODE REVIEW

File: config/amsterdam.config.js [hunk 1]
```diff
@@ -54,7 +56,15 @@ export default {
             "longitude": 4.8952
         },
         "zoom": 12,
-        "attribution": true,
-        "extent":  [ 4.7283, 52.2771, 5.0799, 52.4324 ]
+        "attribution": false,
+        "extent":  [ 4.7283, 52.2771, 5.0799, 52.4324 ],
+        "zoomposition": "bottomright"
+    },
+    "classnames": {
```
Comment (boris-arkenaar) [inline config/amsterdam.config.js:63]: You're still using this?

File: test/index.html [hunk 2]
```diff
@@ -13,7 +14,13 @@
       .loading {
         opacity: 0.3;
       }
-    
+      .embed-search {
+        width: 326px;
+        z-index: 1000000;
```
Comment (boris-arkenaar) [inline test/index.html:19]: That's quite a `z-index`!

**Review by boris-arkenaar — COMMENTED** [review]
`dist/amaps.iife.js.map` mag nog uit de repo verwijderd worden.

File: config/amsterdam.config.js (hunk 1 above)
Comment (ghost) [reply config/amsterdam.config.js:63]: yes, these are additions especially for the Amsterdam viewer.

**Comment by ghost** [comment]
> dist/amaps.iife.js.map mag nog uit de repo verwijderd worden.

Ik heb het verwijderd en dat gepushed. Ik zie nu geen `dist` map in de repo meer? https://github.com/webmapper/amaps/tree/load-amsterdam-style

**Review by boris-arkenaar — APPROVED** [review]

**Pull request merged**